README.rst

```diff
@@ -83,8 +83,9 @@ Authors
 `Kanav Setia <https://github.com/kanavsetia>`__ (Dartmouth),
 `Hannah Sim <https://github.com/hsim13372>`__ (Harvard),
 `Mark Steudtner <https://github.com/msteudtner>`__  (Leiden University),
-`Wei Sun <https://github.com/Spaceenter>`__ (Google) and
-`Fang Zhang <https://github.com/fangzh-umich>`__ (University of Michigan).
+`Wei Sun <https://github.com/Spaceenter>`__ (Google),
+`Fang Zhang <https://github.com/fangzh-umich>`__ (University of Michigan) and
+`Qiming Sun <https://github.com/sunqm>`__ (Caltech).
```

Comment:
Please put yourself in alphabetical order. So you should go before Wei Sun

Reply:
Also add to NOTICE

Reply:
Authors in NOTICE are not in alphabetical order, should I change the order?

Reply:
there is an alphabetical section and and a not-alphabetical section.